src/common/transformations/src/transformations/transpose_sinking/ts_general.cpp

```diff
@@ -77,6 +78,7 @@ bool TSGeneral::run_on_model(const std::shared_ptr<ov::Model>& f) {
         manager.register_pass<TSGeneralBackward>();
```

Comment:
Should we disable ShapeOf here? in Backward transformations batch I mean

Reply:
I can see that DisableShapeOfConstantFolding is registered at line 76.

Reply:
I mean do we actually need it? or should we run it once in Forward transformations?